includes/application.php

```diff
@@ -1,4 +1,5 @@
 <?php
+	$APPLICATION_DIR = $APPLICATION_DIR ?? str_replace('includes', '', dirname(__FILE__));
 	spl_autoload_register(function ($class_name) {
 		global $APPLICATION_DIR;
```

Comment:
@MehaKaushik Checked with the documentation. This is a function and therefore braces should remain as it is. I have incorporated the changes for all methods and classes, and verified it with the php code sniffer.